src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md

```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
 
 You can now run the Galasa CLI too from any directory in your file system without having to specify the absolute path.
 
+## Installing Java 
+
+Install a Java version 11 JDK or later. We do not currently support Java 17 or later. The following example uses Homebrew to install Java version 11 on a MacOS.
+
+1. Install Java version 11 JDK using HomeBrew by running the following command in your terminal:
+```
+brew install openjdk@11
```

Comment:
Brew is a mac-only thing. What about windows or linux users ?

Do we have to tell them how to install it ? or should we do the work and tell them how to do it for all 3 environments ?

Reply:
see previous comment